backend/scripts/cases/verify.py

```diff
@@ -0,0 +1,54 @@
+"""Simple script to verify the csv files in the data folder"""
+from typing import Any, Dict, Set
+
+from data_load import data_load
+
+
+def validate_refs(
+    entities: Dict[str, Any], entity_name: str, entity_key: str, primary_set: Set[str]
+):
+    print()
+    print(f"Checking {entity_name}:")
+    referenced_cases = set([e[entity_key] for e in entities.values()])
+    unknown_cases = referenced_cases - primary_set
+    orphaned_cases = primary_set - referenced_cases
+    print(f"  Found {len(referenced_cases)} referenced {entity_key}s.")
```

Comment:
Yeah, this looks a lot like what I'm doing. The pattern I've been following is keeping a container to store errors, so that when I get to the end of each validation step I can quickly check if the container is falsey because I've not found any errors to add to it...

Reply:
Good to know that its following a pattern - I think I'll leave following the reset to when I integrated this into a backend endpoint